tests/features/common/kie-common.feature

```diff
@@ -33,7 +34,7 @@ Feature: RHPAM and RHDM common tests
       | AUTH_LDAP_SEARCH_TIME_LIMIT                   | 100                          |
       | AUTH_LDAP_USERNAME_BEGIN_STRING               | USER                         |
       | AUTH_LDAP_USERNAME_END_STRING                 | ENDUSER                      |
-    Then file /opt/eap/standalone/configuration/standalone-openshift.xml should contain <login-module code="RealmDirect" flag="optional">
+    Then file /opt/eap/standalone/configuration/standalone-openshift.xml should contain <login-module code="RealmDirect" flag="required">
```

Comment:
we should not change other login modules than ldap one.

Reply:
yep.. sorry, my bad.